tests/test_routes.py

```diff
@@ -571,6 +572,70 @@ def test_update_shopcart_item_with_none_shopcart(self):
             )
             self.assertEqual(res.status_code, status.HTTP_404_NOT_FOUND)
 
+ 
+    def test_clear_empty_shopcart(self):
+        """ It should clear the items but not delete the shopcart """
+        shopcart = self._create_an_empty_shopcart(1)[0]
+        res = self.client.put(
+            f'{BASE_URL}/{shopcart.id}/clear'
+            )
+        logging.debug(res)
+<<<<<<< HEAD
+        self.assertEqual(res.status_code, status.HTTP_200_OK) #was the PUT rqst to clear successful
+=======
+        self.assertEqual(res.status_code, status.HTTP_200_OK) # was the PUT rqst to clear successful
+>>>>>>> 19717293c8270deac12d41f136345172e0c117c5
+        logging.debug(res.status_code)
```

Comment:
could be removed because it is already asserted at line 584 that res.status_code is 200